Nighlty linkcheck failed unexpectedly on Web Archive URL

## Expected behavior

The nightly run of `make docs-linkcheck` succeeds unless a URL is broken. (The job was introduced in #119.)

## Actual behavior

It failed, see [**linkcheck #170**][error].

## Steps to reproduce the problem

- Re-building the nightly job may reproduce the error ([**linkcheck #170**][error])

## Additional information

The cause of [the error][error] is not obvious to me:

```
Warning, treated as error:
/root/project/docs/getting_the_most_out_of_securedrop.rst.rst:86:broken link: https://web.archive.org/web/20201114212453/https://specialprojectsdesk.com/tell-on-trump-1792401813 (404 Client Error: Not Found for url: https://web.archive.org/web/20201118014447/https://specialprojectsdesk.com/tell-on-trump-1792401813)
make: *** [Makefile:17: docs-linkcheck] Error 2
```

Note the dates: `20201114212453` in the URL [that's the URL in the docs][link], and `20201118014447` in the URL that reports the `404 Client Error`.

I can't reproduce the error from a browser, I get a consistent HTTP 200 for the former URL. (The latter does indeed respond with HTTP 404, but I don't know yet how 'linkcheck' got there, I see no redirects in my browser logs.)

  [error]: https://app.circleci.com/pipelines/github/freedomofpress/securedrop-docs/174/workflows/218c1ca4-63a8-459a-9cfa-a4f3f021b43c/jobs/170
  [link]: https://github.com/freedomofpress/securedrop-docs/pull/118/files#diff-36ecf56ad6fe94f1f43551c7090ba8d17dbaf0faa64c28d3f96b8525a388478bR94

--- a/docs/getting_the_most_out_of_securedrop.rst
+++ b/docs/getting_the_most_out_of_securedrop.rst
@@ -91,7 +91,7 @@ proof of concept Twitter advertisement aimed at EPA and NOAA employees to show
 how it can be done. You can read about `how you can do the same thing here`_.
 
 .. _`recently targeted online advertisements`: https://www.wsj.com/articles/gizmodo-ads-target-potential-trump-leakers-1487191482
-.. _`tell on trump`: https://web.archive.org/web/20201114212453/https://specialprojectsdesk.com/tell-on-trump-1792401813
+.. _`tell on trump`: https://web.archive.org/web/20200926063152/https://specialprojectsdesk.com/tell-on-trump-1792401813
 .. _`how you can do the same thing here`: https://freedom.press/news/we-targeted-securedrop-ad-potential-whistleblowers-trump-administration-you-can-too/
 
 Put an Advertisement in Your Physical Paper

